### DISCUSSION TITLE
Plan for a GUI

### DISCUSSION THREAD

**Discussion by Simplegram** [imartinez/privateGPT#354] [General]
Any plan to develop a GUI like Gradio for privateGPT? I want to use this outside my home because you know how mobile a desktop computer is. Thanks.

**Comment by SilentKnightOwl** [comment]
It's not a GUI, but I just ssh to my computer through my router VPN server, which does the same thing.

**Comment by pseudotensor** [comment]
@Simplegram FYI https://github.com/h2oai/h2ogpt has a UI and it uses gradio, so server can be remote and you can access via gradio live link anywhere in work, and there is API client access too.

**Comment by itblogpros** [comment]
I added gradio here if you want to try this
https://github.com/imartinez/privateGPT/discussions/456
